### ISSUE TITLE
Error on Game startup : Method not allowed

### ISSUE THREAD

**Issue by tonym128** [criticalstack/quake-kube#23]
This is an amazing project and I was super keen to give it a go.

I was hoping I could get some help here, since I am very new to kubernetes.

Systes:
Windows 10 64-bit machine 
Latest updates applied
WSL2 and Docker.

I managed to setup the example stack and it is up and running and the websites are available.

However after the name selection and acceptance for the game startup I get an error message : Method not allowed.

I tried in Windows Firefox & Chrome and Mobile Firefox, getting the following :
![image](https://user-images.githubusercontent.com/1162997/99943010-45300600-2d79-11eb-9a07-054f0bb3b1ba.png)

Did I miss something in the setup ?

I will give a bit more of a hack at this week and potentially answer my own issue, but if you know what it is, it would be really appreciated if you could share.

Thanks again for even doing this to start with, I'm finally starting on my journey to learning kubernetes, thanks to you 😄

**Comment by ChrisRx** [comment]
That is really awesome to hear about starting your Kubernetes learning journey!

Unfortunately, I believe I introduced a bug in #21, causing it to fail to get the asset `linuxq3ademo-1.11-6.x86.gz.sh`.  QuakeJS needs this to run but only needs to download it once, which is why I wasn't able to reproduce the error you are receiving until I cleared the site cache and tried again.  I'm working up a fix locally with some tests, should be an easy fix :crossed_fingers:
